core/akka-runtime/src/main/scala/com/ing/baker/runtime/akka/defaultinteractions/TimerInteraction.scala

```diff
@@ -0,0 +1,17 @@
+package com.ing.baker.runtime.akka.defaultinteractions
+
+import cats.effect.{IO, Timer}
+
+import scala.concurrent.duration.FiniteDuration
+
+class TimerInteraction(skipWait: Boolean)(implicit timer: Timer[IO]) {
+
+  class TimeWaited()
+
+  def apply(WaitTime: FiniteDuration): IO[TimeWaited] = {
```

Comment:
I wonder if two implementations are necessary, can't we reduce code duplication?

Reply:
Your are right, I was planning to create a specific one for the Akka runtime based on the schedular of Akka, but using the schedular of cats seems fine so I will remove the duplicates.